recipes/firefox/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/bin
+
+if [ $(uname) == Linux ]; then
+  mv * ${PREFIX}/bin
```

Comment:
We should do the same thing os in the macOS build. Move these files to a folder and add an "entry point" script to call the binary. It is "cleaner" than dumping everything on `bin`. (Maybe we should do that even for Windows with a `.bat` file.)